sdk/search/search/README.md

```diff
@@ -65,12 +65,13 @@ Alternatively, you can get the endpoint and Admin Key from the resource informat
 Once you have an Admin Key, you can use it as follows:
 
 ```js
-const { SearchIndexClient, SearchApiKeyCredential } = require("@azure/search");
+const { AzureKeyCredential } = require("@azure/core-auth");
```

Comment:
`@azure/core-auth` isn't user-facing, it'd be better to re-export from `@azure/search`.

Reply:
Main reason to not have users touch `@azure/core-auth` directly is because it'd then cause confusion between that and `@azure/identity`

Reply:
So the desired experience for customers using both Search and TextAnalytics is to `import * as search from ...; import  * as ta from ...;` and then have both `search.AzureKeyCredential` and `ta.AzureKeyCredential`?

Reply:
That's a good question.  Ideally they'd be importing that type from one library but I don't think `@azure/core-auth` is necessarily the best place for that based on how we've been doing things in the past.  @willmtemple, what are your thoughts on that?

Reply:
I would say the desired experience is grab AzureKeyCredential from whichever is most convenient and use it everywhere it's needed. I agree we don't want to be importing directly from core-auth.

Reply:
Consumers can also alias imports e.g. 

```ts
import { AzureKeyCredential as TACredential } from "@azure/ai-text-analytics";
```